pkgs/development/ocaml-modules/riot/default.nix

```diff
@@ -1,33 +1,42 @@
 { lib
 , bigstringaf
 , buildDunePackage
+, cstruct
 , fetchurl
 , iomux
+, mdx
+, poll
 , ptime
 , telemetry
 , uri
 }:
 
 buildDunePackage rec {
   pname = "riot";
-  version = "0.0.5";
+  version = "0.0.7";
 
   minimalOCamlVersion = "5.1";
 
   src = fetchurl {
     url = "https://github.com/leostera/riot/releases/download/${version}/riot-${version}.tbz";
-    hash = "sha256-Abe4LMxlaxK3MVlg2d8X60aCuPGvaOn+4zFx/uH5z4g=";
+    hash = "sha256-t+PMBh4rZXi82dUljv3nLzZX5o1iagBbQ9FfGnr/dp4=";
   };
 
   propagatedBuildInputs = [
     bigstringaf
+    cstruct
     iomux
```

Comment:
Is `iomux` still needed?

Reply:
Good catch! iomux is not needed anymore. Thanks!